public/js/machine.js

```diff
@@ -1 +1,37 @@
-$('#machine-table').tablesort();
+// $('#machine-table').tablesort();
+function generateCalendar(calendar) {
+  let eventDates = [];
+  const bucket = 0;
```

Comment:
Not having these seemingly arbitrary indices would be the benefit of using an array of objects instead of an array of arrays as mentioned above.